arrow-libs/resilience/arrow-resilience/build.gradle.kts

```diff
@@ -6,8 +6,7 @@ kotlin {
   sourceSets {
     commonMain {
       dependencies {
-        api(projects.arrowCore)
-        implementation(libs.coroutines.core)
+        api(projects.arrowFxCoroutines)
```

Comment:
Same concern as with #3794, are we OK depending on `fx-coroutines` in the `resilience` module? @nomisRev

Reply:
Alas, I've realized that moving `bracket` to `arrow-exception-utils` would require adding a dependency on `kotlinx.coroutines` for that module, and transitively to `arrow-core` and `arrow-autoclose`. And this is something we don't want to do, at least at this point.